compiler-rt/test/memprof/TestCases/default_options.cpp

```diff
@@ -1,5 +1,8 @@
 // RUN: %clangxx_memprof -O2 %s -o %t && %run %t 2>&1 | FileCheck %s
 
+// Check that overriding __memprof_default_options() takes precedence over the LLVM flag
+// RUN: %clangxx_memprof -O2 %s -o %t-flag -mllvm -memprof-runtime-default-options="verbosity=0 help=0" && %run %t-flag 2>&1 | FileCheck %s
+
 const char *kMemProfDefaultOptions = "verbosity=1 help=1";
 
 extern "C" const char *__memprof_default_options() {
```

Comment:
I think the CHECK statement below will need to be updated to check `verbosity=1 help=1` to verify the behaviour.

Reply:
`Available flags for MemProfiler:` will only output if `help=1`, which is checked on line 6. So this indeed verifies the expected behavior.

Reply:
Thanks for pointing that out.